src/Model/FileGroup.php

```diff
@@ -132,4 +133,25 @@ public function unsetFile(SplFileInfo $file): void
             }
         }
     }
+
+    public function serialize(string $searchDirectory): object
+    {
+        $fileGroup = new StdClass();
```

Comment:
Can you make a model instead of using \stdClass?

Reply:
I will let `FileGroup` implement `JsonSerializable` instead, that way we don't need `stdClass` nor a model class.